### ISSUE TITLE
Can't modify messages.yml without stopping the server

### ISSUE THREAD

**Issue by goldenlib** [stefvanschie/buildinggame#456]
## Issue

__I'm using version:__ *12.7.1*  
__I'm on Minecraft:__ *1.21*  
__I'm running:__ *Paper*

__Seen results:__  
When I reload the plugin, it reset the modification I made in messages.yml

__Expected results:__  
I was hoping that I could modify the file, and it would reload with the configuration to allow me to see the change immediately.

__How to reproduce:__  
Doing any change in this file seems to be deleted when I reload the plugin. The only way to modify the file is to turn the server off and modify. Else, the plugin removes the changes

**Comment by stefvanschie** [comment]
The plugin should detect and apply changes to the messages.yml automatically. Could you indicate which OS your server is running on? You could also try enabling the `debug` option in the config.yml. If the messages.yml is reloaded because of changes being made, your console log will have the following message:
```
Detected changes in the messages.yml
```
If this does not happen, or happens when you haven't made any changes, please also let me know.

**Comment by goldenlib** [comment]
I'm on Windows 10. I found this while I was testing the plugin on my local server on my computer with jdk-21. I turned debug mod on and there is no way to see _"Detected changes in the messages.yml"_ even when I reload the plugin. I tried multiple way but every time, I reload Notepad++ tell me the file was modified by another program, and it rolls back like it was before. The weird thing is that it doesn't happen on the config file. Every second, it seems the plugin write on the file and put it like it's in his memory (Windows explorer tells me last time the file was modified is now).
